Project.toml

```diff
@@ -5,12 +5,10 @@ version = "0.10.0"
 [deps]
```

Comment:
maybe bump package version for a release?

Reply:
I'll do that directly on master after merging.